lib/file.js

```diff
@@ -3,57 +3,58 @@ var path  =  require('path')
  ,  _   =  require('underscore');
 
 var markdownExts = ['.md', '.markdown'];
+var mdxExts = ['.mdx'];
 var ignoredDirs  = ['.', '..', '.git', 'node_modules'];
 
-function separateFilesAndDirs(fileInfos) {
+function separateFilesAndDirs(fileInfos, syntax = "md") {;
```

Comment:
Can you validate the default value is actually needed for all places you're doing `syntax = "md"`? If we're always passing in a `syntax` variable, the default is not-DRY noise.